pkg/csi/service/common/prometheus_metrics_helper.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+const (
+	// PrometheusBlockVolumeType represents the block volume type.
+	PrometheusBlockVolumeType = "block"
+	// PrometheusFileVolumeType represents the file volume type.
+	PrometheusFileVolumeType = "file"
+	// PrometheusUnknownVolumeType is used in situation when the volume type could not be found.
+	PrometheusUnknownVolumeType = "unknown"
+
+	// PrometheusCreateOpType represents the CreateVolume operation.
+	PrometheusCreateOpType = "create"
```

Comment:
Can we change this to "createvolume" and "deletevolume"?
later if we have some other create operations such as snapshot etc we can distinguish them.

Reply:
Makes sense.